package.json

```diff
@@ -57,6 +57,7 @@
     "pretty-ms": "7.0.0",
     "semver": "7.3.2",
     "serialised-error": "1.1.3",
+    "tough-cookie": "3.0.1",
```

Comment:
There is a v4 available, but since postman-runtime was using 3.0.1 I decided to keep it same as that.